### PR TITLE
Use 'istioctl dashboard' instead of port-forward

### PR DESCRIPTION
**Pull request by esnible**
Mitigates https://github.com/istio/istio.io/issues/4954

Note that the original request was for the dashboard docs, which this doesn't fix.

@geeknoid Currently `istioctl dashboard` picks a random local port.  This avoids conflicts, but it makes it impossible to directly link to the localhost URL in the documentation.  To finish this work item I am considering a change so that `dashboard` attempts to grab the well-known port number, and only picks a random one if the port is already busy.  That would minimize the doc changes.  What do you think?

### CODE REVIEW

File: content/en/docs/ops/common-problems/observability-issues/index.md [hunk 1]
```diff
@@ -174,10 +174,10 @@ in processing the most recent configuration as supplied.
 1. In Kubernetes environments, setup port-forwarding with:
 
     {{< text bash >}}
-    $ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 &
+    $ istioctl dashboard prometheus &
     {{< /text >}}
 
-1. Visit `http://localhost:9090/targets`
+1. In the Prometheus browser window, select Status then Targets
```
[the remark this answers is not in the document]
Comment (howardjohn) [reply content/en/docs/ops/common-problems/observability-issues/index.md]: Below we specify to go to :9090 again

File: content/en/docs/ops/common-problems/observability-issues/index.md [hunk 2]
```diff
@@ -174,10 +174,10 @@ in processing the most recent configuration as supplied.
 1. In Kubernetes environments, setup port-forwarding with:
 
     {{< text bash >}}
-    $ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 &
+    $ istioctl dashboard prometheus &
```
Comment (howardjohn) [inline content/en/docs/ops/common-problems/observability-issues/index.md]:
```suggestion
    $ istioctl dashboard prometheus
```

For consistency - no strong opinion on this though

File: content/en/docs/tasks/observability/distributed-tracing/jaeger/index.md [hunk 3]
```diff
@@ -65,10 +63,10 @@ Open your browser to [http://localhost:15032](http://localhost:15032).
 
 ## Cleanup
 
-1.  Remove any `kubectl port-forward` processes that may still be running:
+1.  Remove any `istioctl dashboard` processes that may still be running:
```
Comment (geeknoid) [inline content/en/docs/tasks/observability/distributed-tracing/jaeger/index.md]:
```suggestion
1.  Remove any `istioctl` processes that may still be running:
```

File: content/en/docs/tasks/observability/distributed-tracing/zipkin/index.md [hunk 4]
```diff
@@ -65,10 +63,10 @@ Open your browser to [http://localhost:15032](http://localhost:15032).
 
 ## Cleanup
 
-1.  Remove any `kubectl port-forward` processes that may still be running:
+1.  Remove any `istioctl dashboard` processes that may still be running:
```
Comment (geeknoid) [inline content/en/docs/tasks/observability/distributed-tracing/zipkin/index.md]:
```suggestion
1.  Remove any `istioctl` processes that may still be running:
```

File: content/en/docs/tasks/observability/metrics/querying-metrics/index.md [hunk 5]
```diff
@@ -108,10 +108,10 @@ docs](https://prometheus.io/docs/querying/basics/).
 
 ## Cleanup
 
--   Remove any `kubectl port-forward` processes that may still be running:
+-   Remove any `istioctl dashboard` processes that may still be running:
```
Comment (geeknoid) [inline content/en/docs/tasks/observability/metrics/querying-metrics/index.md]:
```suggestion
-   Remove any `istioctl` processes that may still be running:
```

**Comment by geeknoid** [comment]
@esnible Do you want to land this PR for 1.4?

**Comment by esnible** [comment]
I have addressed the comments and fixed the conflict.

Remaining is improving deep links to Grafana.  I propose waiting for https://github.com/istio/istio/pull/18552 to merge and then doing a follow-up.

**Pull request merged**